libsg3/ses/ses.pyx

```diff
@@ -329,11 +329,12 @@ cdef class EnclosureDevice(object):
         }
         return element_dict[index]
 
-    def decode_response(self, buff):
+    cdef str decode_response(self, char * buff):
+        cdef bytes py_bytes = buff
         try:
-            return bytes(buff, encoding='ascii').decode()
-        except Exception:
-            return ""
+            return py_bytes.decode(errors='replace')
```

Comment:
Is `encoding='ascii'` the default?

Reply:
I don't mean it does not work.  I mean what specification says we should receive here and whether we should pass any non-ASCII though.  Though may be not a problem to be more relaxed.